.goreleaser.yml

```diff
@@ -14,7 +14,7 @@ builds:
 - main: ./nats-streaming-server.go
   binary: nats-streaming-server
   ldflags:
-    - -w -X github.com/nats-io/nats-streaming-server/server.gitCommit={{.ShortCommit}} -X github.com/nats-io/nats-server/v2/server.gitCommit=bb87a650
+    - -w -X github.com/nats-io/nats-streaming-server/server.gitCommit={{.ShortCommit}} -X github.com/nats-io/nats-server/v2/server.gitCommit=7cf537e
```

Comment:
This supposed to be 8 chars?

Reply:
I use whatever is being reported when starting the server. In the failed 2.7.0, it was 1 character more, but it looks like 2.7.1 is back to 7 characters.
I downloaded the NATS Server release and start the server and it prints 7cf537e.